azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -369,10 +367,9 @@ internal class CallCompositeActivity : AppCompatActivity() {
                 ActivityInfo.SCREEN_ORIENTATION_FULL_SENSOR
             CallCompositeSupportedScreenOrientation.USER_LANDSCAPE ->
                 ActivityInfo.SCREEN_ORIENTATION_USER_LANDSCAPE
-            null -> null
             else -> {
                 logger.warning("Not supported screen orientation")
-                null
+                ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
```

Comment:
This will not be good for CallScreen
It is better to return nil if not possible to convert.